sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -129,7 +129,7 @@ public partial class SearchOptions
         public double? MinimumCoverage { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> OrderBy { get { throw null; } }
         public Azure.Search.Documents.Models.QueryAnswer? QueryAnswer { get { throw null; } set { } }
-        public uint QueryAnswerCount { get { throw null; } set { } }
+        public uint? QueryAnswerCount { get { throw null; } set { } }
```

Comment:
We should change this to `int`.  We generally don't use `uint` to enforce positive values.